css/styles.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Regular.ttf);
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Bold.ttf);
+    font-weight: bold;
+}
+
+body {
+    color: #424242;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 14px;
+    letter-spacing: .01em;
+    line-height: 1.3;
+    padding: 24px;
+    max-width: 800px;
+}
+
+@media screen and (max-width: 960px) {
+    body {
+        padding: 16px;
+    }
+}
+
+h1 {
+    font-size: 1.5em;
+    margin: .67em 0;
+}
+
+h2 {
+    font-size: 1.17em;
+}
+
+h3 {
+    font-size: 1em;
+}
+
+a {
+    color: #996189;
+    text-decoration: underline;
+    cursor: pointer;
+    padding: 0;
+    display: inline;
+}
+
+img {
+    max-width: 100%;
+}
+
+button {
```

Comment:
Does it make sense to use the tag or set a .button class for this? Given the wide array of content we might be pulling in from, not sure how safe it is to automatically style all buttons. I don't have a strong preference on this, just something to think about

Reply:
I think that's a valid point! Changed to a `.button` class as you suggested.